tests/scripts/thread-cert/Cert_5_1_02_ChildAddressTimeout.py

```diff
@@ -97,21 +96,13 @@ def test(self):
         time.sleep(5)
         for addr in ed_addrs:
             if addr[0:4] != 'fe80':
-                try:
-                    self.nodes[LEADER].ping(addr)
-                    self.fail()
-                except pexpect.TIMEOUT:
-                    pass
+                self.assertEqual(self.nodes[LEADER].ping(addr), False)
```

Comment:
I wasn't aware it existed. This is my first time with python tests. I can make the changes if you would like.